.ci/gcb-push-downstream.yml

```
@@ -31,40 +31,44 @@ steps:
 
     # TPG
     - name: 'gcr.io/graphite-docker-images/bash-plus'
```

Do these need to use `go-plus`? That is what the previous PR did.

```suggestion
    - name: 'gcr.io/graphite-docker-images/go-plus'
```